### PR TITLE
Migrate to slog in static analysis, and remove now-dead logging code.

### PR DESCRIPTION
**Pull request by calebbrown**

### CODE REVIEW

File: sandboxes/staticanalysis/staticanalyze.go [hunk 1]
```diff
@@ -220,7 +225,7 @@ func run() (err error) {
 func main() {
 	err := run()
```
Comment (maxfisher-g) [inline sandboxes/staticanalysis/staticanalyze.go:226]: self-nit: can we make this `if err := run(); err != nil {  ...` please?

I actually did this locally but hadn't pushed the change yet.

File: internal/log/log.go [hunk 2]
```diff
@@ -62,7 +45,7 @@ func DefaultLoggingEnv() LoggingEnv {
 // "false" will use the default development configuration.
 //
 // Note: this method MUST be called before any other method in this package.
-func Initialize(env string) *zap.SugaredLogger {
+func Initialize(env string) {
 	var err error
 	var logger *zap.Logger
```
Comment (maxfisher-g) [inline internal/log/log.go:50]: Do we still need the zap logger now that we're using slog for everything?

**Review by maxfisher-g — APPROVED** [review]

File: internal/log/log.go (hunk 2 above)
Comment (calebbrown) [reply internal/log/log.go:50]: Yes, for now. We are using it as the handler for `slog`, including a StackDriver library for sending logs to GCP.

File: sandboxes/staticanalysis/staticanalyze.go (hunk 1 above)
Comment (calebbrown) [reply sandboxes/staticanalysis/staticanalyze.go]: Done

**Pull request merged**